src/main/java/bio/terra/service/filedata/azure/util/BlobCrl.java

```diff
@@ -109,8 +114,35 @@ public BlobContainerCopier createBlobContainerCopier(URL sourceUrl, String desti
    *
    * @param blobName blob name to delete.
    */
-  public void deleteBlob(String blobName) {
-    blobContainerClientFactory.getBlobContainerClient().getBlobClient(blobName).delete();
+  public boolean deleteBlob(String blobName) {
+    try {
+      blobContainerClientFactory.getBlobContainerClient().getBlobClient(blobName).delete();
+      return true;
+    } catch (BlobStorageException e) {
+      if (e.getStatusCode() == HttpStatus.NOT_FOUND.value()) {
+        return false;
+      } else {
+        throw new PdaoException("Error deleting file", e);
+      }
+    }
+  }
+
+  /**
+   * Deletes the specified blob in the container. Logs on failure but does not throw error.
+   *
+   * @param blobName blob name to delete.
+   */
+  public boolean deleteBlobQuietFailure(String blobName) {
```

Comment:
Is there a need to return a value here? The result is ignored.

Reply:
True, I'll switch to void.